css/style.css

```
@@ -33,13 +35,96 @@
     line-height: 26px;
     font-style: normal;
     color: var(--color-black);
-    padding: 0;
-    margin: 0;
+    background-color: var(--color-white);
+    text-align: center;
+
+    display: grid;
+    grid-template-rows: min-content 1fr min-content;
+    min-height: 100%;
+    align-content: start;
+}
+
+  /*Grid*/
+
+  .page {
+
+    height: 100%;
+}
+
+  .container {
+
+    display: grid;
+    margin: 0 auto;
     max-width: 1200px;
+    background: #FFFFFF;
+    box-shadow: 0px 5px 15px rgba(0, 1, 1, 0.2);
+}
+
+  .advantages-list {
+
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    background-color: var(--background-features);
+}
+
+  .advantages-item:first-child {
+
+    display: grid;
+    grid-column: 1 / -1;
+    grid-template-columns: 1fr 2fr;
+}
+
+  .advantages-item:nth-child(2) {
+
+    display: grid;
+    grid-column: 1 / -1;
+    grid-template-columns: 2fr 1fr;
+}
+
+  .feature-items {
+
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-column: 1 / -1;
     background-color: var(--color-white);
+  
+}
+
+  .feature-item {
+
+    padding: 50px 0;
+}
+
+  .main-footer {
+
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
     text-align: center;
 }
 
+  .social {
+
+    display: block;
+    width: 46px;
+    height: 48px;
+    justify-content: center;
+    align-items: center;
+}
+
+  .footer-social-list {
+
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+}
+
+  
+
+  .gray {
+
+   padding: 50px 0px ;
```

этих классов может быть много на странице, лучше увеличить специфичность селектора:
```suggestion
  .advantages-item.gray {
   padding: 50px 0px;
```